spec/factories/users.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :user, aliases: [:member] do
-    github_handle { FFaker::Name.name.underscore }
+    github_handle { FFaker::Name.name.parameterize }
```

Comment:
Makes 💯 sense as the smallest possible change to put things straight here, fully agree 👍 

I didn't even know the user factory looks like this...

However, for making things even simpler and less likely to cause flaky specs, I'd even go for something like
```ruby
github_handle { FFaker::Internet.user_name }
```
What do you think?

Reply:
Thank you @klappradla!

I tried to find something suitable in FFaker, but nothing seemed to be perfect...

`FFaker::Internet.user_name`, for example generates all these:

```
jewell.towne
wilda
dorthea_reichert
```

From these, I think only wilda is a valid Github username, and it won't generate other valid usernames like `rails-girls`.

Closest I found was `FFaker::InternetSE.login_user_name` which seems to only return simple words or words with underscores, but even underscores seem to be invalid for Github ([see here](https://github.com/shinnn/github-username-regex) - this is likely not very reliable, but I couldn't find any username with a `_` after a quick search).

🤷‍♀️ wish we could use something directly from FFaker but I can't find anything that would match Github's usernames...

Unless we use `FFaker::InternetSE.user_name_variant_short` which only returns single words - that would work. Not sure if it's much clearer though - what do you think?

Reply:
Hey all! Hope you don't mind my throwing in an opinion. I think the `user_name_variant_short` is a good compromise.

Reply:
It's much appreciated @ramonh! 🙌

I have the commit with the change ready to push. I'll give @klappradla a chance to chime in as well before I do, though.

Reply:
As said @pgaspar: I'd even merge it as it is, was just a cosmetic comment.

If you want, feel fee to update, otherwise just give me you "go" and I merge this.

Reply:
Pushed! Thank you :shipit: